pkg/apis/core/validation/validation.go

```diff
@@ -1651,8 +1651,8 @@ var allowedTemplateObjectMetaFields = map[string]bool{
 	"Labels":      true,
 }
 
-// PersistentVolumeSpecValidationOptions contains the different settings for PeristentVolume validation
-type PersistentVolumeSpecValidationOptions struct {
+// PersistentVolumeValidationOptions contains the different settings for PeristentVolume validation
+type PersistentVolumeValidationOptions struct {
```

Comment:
why was this type renamed? the options in it only apply to spec, right?

Reply:
The intention was to validate the new status field as well, but we dropped it along the way and this commit got forgotten - dropping it as well as the rename is not needed now.